ui/src/app/pages/homepage/homepage.tsx

```diff
@@ -367,30 +367,36 @@ export const Homepage = withUserProfile()(class extends React.Component<
                                             dataUseAgreementCompleted={dataUseAgreementCompleted}/>
                       ) : (
-                        <FlexRow style={{paddingTop: '2rem'}}>
-                          <FlexColumn style={styles.contentWrapperLeft}>
-                            <FlexRow style={styles.mainHeader}>Researcher Workbench</FlexRow>
-                            <CardButton onClick={() => navigate(['workspaces/build'])}
-                                        style={{margin: '1.9rem 106px 0 3%'}}>
-                              Create a <br/> New Workspace
-                              <ClrIcon shape='plus-circle' style={{height: '32px', width: '32px'}}/>
-                            </CardButton>
-                          </FlexColumn>
-                          <FlexColumn style={styles.contentWrapperRight}>
-                            <a onClick={() => navigate(['workspaces'])}
-                               style={{fontSize: '14px', color: colors.primary}}>
-                              See All Workspaces</a>
-                            <FlexColumn style={{marginRight: '3%'}}>
-                              <div style={{color: colors.primary, height: '1.9rem'}}>
-                                <div style={{marginTop: '.5rem'}}>Your Last Accessed Items</div>
-                              </div>
-                              <RecentWork dark={true}/>
+                          <React.Fragment>
+                            <FlexColumn>
+                              <FlexRow style={{justifyContent: 'space-between', alignItems: 'center'}}>
+                                <FlexRow style={{alignItems: 'center'}}>
+                                  <SmallHeader style={{marginTop: '0px'}}>Workspaces</SmallHeader>
+                                  <ClrIcon
+                                    shape='plus-circle'
+                                    size={21}
+                                    className={'is-solid'}
+                                    style={{color: colors.accent, marginLeft: '1rem', cursor: 'pointer'}}
+                                    onClick={() => navigate(['workspaces/build'])}
+                                  />
+                                </FlexRow>
+                                <span
+                                  style={{alignSelf: 'flex-end', color: colors.accent, cursor: 'pointer'}}
+                                  onClick={() => navigate(['workspaces'])}
+                                >
+                                  See all Workspaces
+                                </span>
+                              </FlexRow>
+                              <RecentWorkspaces />
                             </FlexColumn>
-                          </FlexColumn>
-                        </FlexRow>)
+                            <FlexColumn>
+                              <SmallHeader>Recently Accessed Items</SmallHeader>
+                              <RecentWork/>
```

Comment:
Is this `RecentAccessedObjects` (e.g., cohorts, concept sets, etc) or `RecentWorkspaces`? If `RecentWorkspaces`, I'd probably lean towards full spelling there. If not, it would probably be good to rename this to be clearer.